### PR TITLE
malicious ration reputation script

### PR DESCRIPTION
**Pull request by erezh31**

### CODE REVIEW

**Comment by erezh31** [comment]
@meirwah plz review

File: Scripts/script-MaliciousRatioReputation.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      }
+      demisto.results({
+          'Type': entryTypes['note'],
+          'EntryContext': ec,
+          'Contents': 'Malicious ratio for %s is %.2f' % (indicator['value'], mr_score),
+          'ContentsFormat': formats['text']
+      })
+type: python
+tags:
+- reputation
```
Comment (meirwah) [inline Scripts/script-MaliciousRatioReputation.yml]: can it be also enhancement script?

**Comment by erezh31** [comment]
@meirwah ready to review. changed to work well as reputation script (and also can used as script part of playbook...)

File: Scripts/script-MaliciousRatioReputation.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                  'Score': 2, #suspicious
+                  'Vendor': 'DBot-MaliciousRatio',
+                  'Indicator': indicator['value']
+              }
+              entry = {
+                  'Type': entryTypes['note'],
+                  'EntryContext': ec,
+                  'Contents': ec['DBotScore']['Score'],
+                  'ContentsFormat': formats['text'],
+                  'HumanReadable': 'Malicious ratio for %s is %.2f' % (indicator['value'], mr_score),
```
Comment (meirwah) [inline Scripts/script-MaliciousRatioReputation.yml]: add here the score result, is suspcious / malicous/ good

Comment (erezh31) [reply Scripts/script-MaliciousRatioReputation.yml]: actually we said the Malicious ratio will set only to suspcious, and the mr_score is the  Malicious ratio itself (value between 0-1)

**Pull request merged**